docs/enterprise-starter.mdx

```diff
@@ -0,0 +1,67 @@
+# Enterprise Starter
+
+<p className="subtitle">Learn about the Enterprise Starter plan tailored for teams and individuals.</p>
+
+<Callout type="info">Enterprise Starter is available for Sourcegraph versions 6.0+ and supports VS Code, JetBrains, Cody Web.</Callout>
+
+The Enterprise Starter plan offers a multi-tenant Sourcegraph designed for small teams and individuals that require the same core features of a traditional Sourcegraph instance but with a simplified management experience. This plan provides a fully managed version of Sourcegraph (AI + code search + omnibox, with privately indexed code) through a self-serve flow.
+
+## Team seats
+
+The Enterprise Starter plan is priced at **$19 per month per seat**. You can add or remove team members at any time. Existing Cody Pro users can also sign up for the Enterprise Starter by paying $19 per seat. However, their Cody Pro subscription will neither be upgraded nor canceled. Instead, they will have two live subscriptions.
+
+## Enterprise Starter team roles
+
+The Enterprise Starter plan includes the following team-level roles:
+
+- **Admin**: Has full access to the workspace, including the ability to manage users, billing, and settings
+- **Member**: Can access repositories in the workspace and can experience the supported features
+
+## Billing
+
+Workspaces on the Enterprise Starter plan are billed monthly based on the number of team seats purchased. In case of overdue or failing payment, there is a grace period during which the workspace admins will receive a daily notification to complete the transaction.
+
+## Features supported
+
+The Enterprise Starter plan supports a variety of AI and search-based features like:
+
+|             **AI features**              |        **Code Search**         |     **Deployment**     |                      **Management**                       |        **Support**        |
```

Comment:
The deployment column doesn't make sense to me, not sure we need it

Reply:
I am waiting on the finalized message from Kelvin. Once that's done I'll have a final copy for it.